vendor/modules.txt

```diff
@@ -63,7 +62,6 @@ github.com/davecgh/go-spew/spew
 github.com/fatih/color
 # github.com/francoispqt/gojay v1.2.13
 ## explicit
```

Comment:
Mmh, this shouldn't be an explicit dependency anymore. Please, re-run `go mod tidy && go mod vendor`.

Reply:
done. somehow i always keep forgetting this :(